Add issues templates for feature requests and bug reports (#589 / #667) (#667)
junit-pioneer/junit-pioneer: .github/issue--bug-report.md

---

> name: 'Issue: Bug report'

Reviewer: Why the "Issue" prefix here and not for FR?

```suggestion
name: 'Bug report'
```

> about: Please use this template if you want to report a bug

Reviewer: Use similar phrasing as for FR:

```suggestion
about: Report a bug for this project
```

title: "[BUG]"
labels: ["🚦 status: in discussion", "🏗️ type: bug"]
assignees: ''

---

**Describe the bug**

Please provide a detailed description for the bug, containing the current and expected behavior as well as code to reproduce.

**Description**

**Current behavior**

**Expected behavior**

**Code to Reproduce**

> Please provide an example code (using the code markdown) to demonstrate the bug in praxis.

Reviewer: ```suggestion
Please provide example code (using markdown) to demonstrate the bug in practice.
```

**Versions**

Please provide information about the versions of the following tools you use, when the bug occurs:

> * Which JUnit Pioneer version do you use, when the bug occurs?

Reviewer: ```suggestion
* Which JUnit Pioneer version do you use?
```
